client/ayon_core/addon/interfaces.py

```diff
@@ -117,77 +147,85 @@ def get_inventory_action_paths(self, host_name):
 
         Args:
            host_name (str): For which host are the plugins meant.
-        """
 
+        """
         return self._get_plugin_paths_by_type("inventory")
 
 
 class ITrayAddon(AYONInterface):
     """Addon has special procedures when used in Tray tool.
 
-    IMPORTANT:
-    The addon. still must be usable if is not used in tray even if
-    would do nothing.
-    """
+    Important:
+        The addon. still must be usable if is not used in tray even if it
+        would do nothing.
 
+    """
     tray_initialized = False
-    _tray_manager = None
+    manager: AddonsManager = None
```

Comment:
Why this was added?

Reply:
because it is used down the line like in `add_doubleclick_callback` and mypy doesn't like that it doesn't have type defined